tests/core/engine_adapter/test_integration.py

```diff
@@ -1898,3 +1898,85 @@ def test_dialects(ctx: TestContext):
     pd.testing.assert_frame_equal(
         df, pd.DataFrame([[1, 1, 1, 1]], columns=expected_columns), check_dtype=False
     )
+
+
+@pytest.mark.parametrize(
+    "time_column, time_column_type, time_column_format, result",
+    [
+        (
+            exp.null(),
+            exp.DataType.build("TIMESTAMP"),
+            None,
+            {
+                "default": None,
+                "bigquery": pd.NaT,
+                "databricks": pd.NaT,
+                "duckdb": pd.NaT,
+                "motherduck": pd.NaT,
+                "snowflake": pd.NaT,
+                "spark": pd.NaT,
+            },
+        ),
+        (
+            "2020-01-01 00:00:00+00:00",
+            exp.DataType.build("DATE"),
+            None,
+            {
+                "default": datetime(2020, 1, 1).date(),
+                "duckdb": pd.Timestamp("2020-01-01"),
+            },
+        ),
+        (
+            "2020-01-01 00:00:00+00:00",
+            exp.DataType.build("TIMESTAMPTZ"),
+            None,
+            {
+                "default": pd.Timestamp("2020-01-01 00:00:00+00:00"),
+                # TODO: Figure out why this is bytes and if this is really ok
+                "tsql": b"\x00\x00\x00\x00\x00\x00\x00\x005\xab\x00\x00\x00\x00\x07\xe0",
+                "mysql": pd.Timestamp("2020-01-01 00:00:00"),
+                "spark": pd.Timestamp("2020-01-01 00:00:00"),
+            },
+        ),
+        (
+            "2020-01-01 00:00:00+00:00",
+            exp.DataType.build("TIMESTAMP"),
+            None,
+            {
+                "default": pd.Timestamp("2020-01-01 00:00:00"),
+                # TODO: Figure out why this has a timezone and if this is really ok
```

Comment:
Databricks' timestamp type is tz-aware: 
"Represents values comprising values of fields year, month, day, hour, minute, and second, with the session local time-zone. The timestamp value represents an absolute point in time." https://docs.databricks.com/en/sql/language-manual/data-types/timestamp-type.html

They are adding a non-aware version `TIMESTAMP_NTZ` that's currently in public preview  - you have to specify a table option to use it: 
"Feature support is enabled automatically when you create a new Delta table with a column of TIMESTAMP_NTZ type. It is not enabled automatically when you add a column of TIMESTAMP_NTZ type to an existing table. To enable support for TIMESTAMP_NTZ columns, support for the feature must be explicitly enabled for the existing table." https://docs.databricks.com/en/sql/language-manual/data-types/timestamp-ntz-type.html

Reply:
Ah ok will update the comment